jobs/tests.py

```diff
@@ -1,3 +1,82 @@
 from django.test import TestCase
+from django.utils import timezone
 
-# Create your tests here.
+from model_mommy import mommy
+
+from jobs.models import Job
+
+
+class JobModelTests(TestCase):
+    """Tests for custom methods of Job model."""
+
+    """Setting up models with mommy."""
```

Comment:
move all docstrings to the line below the function definition (just the standard convention)